timingApp/Db/event_log.db

```diff
@@ -205,6 +205,19 @@ record(compress, "$(P)$(R)SUBSECbuffer"){
   field(FLNK, "$(P)$(R)TSbuffer")
 }
 
+record(aai, "$(P)$(R)bufferLABELS"){
+  field(ASG, "Reserved")
+  field(NELM, 2)
+  field(FTVL, "STRING")
+  field(INP, {const:["Timestamp", "Event"]})
+  info(Q:group, {
+    "$(P)$(R)EventTable-Mon": {
+      +id: "epics:nt/NTTable:1.0",
```

Comment:
Where does this identifier come from?

Reply:
From the NTTable example in the PVXS docs.

I don't know if there's a centralized listing for these.

Reply:
Okay. But what I don't get whether this pattern has any meaning at all. It seems to be [somewhat arbitrary](https://epics-base.github.io/pvxs/qgroup.html#json-reference), but I didn't find any explicit note of that.

Reply:
> But what I don't get whether this pattern has any meaning at all.

It does, same as "epics:nt/NTScalar(Array):1.0".

It's just... not documented anywhere. If you use  `+type: "scalar"`, PVXS will use "epics:nt/NTScalar:1.0" for that value, for example.

You can find them in nt.cpp: https://github.com/search?q=repo%3Aepics-base%2Fpvxs%20epics%3Ant&type=code

Reply:
Got it. So, it turns out that we are really specifying the type here. I thought it was some kind of identifier for the group instance itself.